feature/lldp/tests/core_lldp_tlv_population_test/core_lldp_tlv_population_test.go

```diff
@@ -99,14 +99,20 @@ func verifyNodeConfig(t *testing.T, nodeTelemetry *device.DevicePath, port *onda
 	if lldpEnabled != state.GetEnabled() {
 		t.Errorf("LLDP enabled got: %t, want: %t.", state.GetEnabled(), lldpEnabled)
```

Comment:
Replace "LLDP enabled" with "LLDP state.GetEnabled()"

Please make similar changes to other Logf() and Error()

Reply:
lldpEnabled in if condition is a bool as TC has 2 cases one with LLDP enabled and one LLDP disabled,  code compares lldEnabled with LLDP state.GetEnabled().
 
 Are you referring if condition or message printed thru Errorf?